addons/binding/org.openhab.binding.meteostick/src/main/java/org/openhab/binding/meteostick/handler/MeteostickBridgeHandler.java

```diff
@@ -75,7 +75,6 @@ public MeteostickBridgeHandler(Bridge thing) {
     @Override
     public void initialize() {
         logger.debug("Initializing MeteoStick Bridge handler.");
-        super.initialize();
 
         updateStatus(ThingStatus.OFFLINE);
```

Comment:
Could you maybe change this to UNKNOWN here, because setting it to OFFLINE in any case seems to be wrong as well?